Slambook/index.js

```diff
@@ -54,30 +55,8 @@ form.addEventListener("submit", (e) => {
 

```

Comment:
Consider using setAttribute to set the class name instead of directly assigning to classList for better compatibility and to avoid overwriting existing classes on the element.